packages/block-library/src/block/edit.js

```diff
@@ -5,7 +5,7 @@ import { useDispatch, useSelect } from '@wordpress/data';
 import {
 	useEntityBlockEditor,
 	useEntityProp,
-	__experimentalUseEntityRecord as useEntityRecord,
+	useEntityRecord as useEntityRecord,
```

Comment:
The `as` is redundant here (and ditto for the other places where this occurs in this PR).